internal/pkg/policy/standalone_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+//go:build !integration
+// +build !integration
```

Comment:
(nitpick) I don't think we need the `// +` directives

Reply:
:+1: Removed from all files.